src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue12642.cs

```diff
@@ -19,12 +18,11 @@ public Issue12642(TestDevice testDevice) : base(testDevice)
 		[Category(UITestCategories.Compatibility)]
 		public void ClickingQuicklyBetweenTopTabsBreaksContent()
```

Comment:
This test is failing on Windows (seems like a crash):

<img width="894" alt="image" src="https://github.com/user-attachments/assets/2dcc07f9-9142-40ea-8531-53a34ff178c7">

Reply:
@jsuarezruiz , we also get this crash occasionally, for now added if defines to ignore this case on Windows as well to investigate more further. Could you please check and let me know if you have any concerns.